workers/scanner.go

```diff
@@ -102,14 +98,25 @@ type (
 )
 
 // NewScanner creates a new Scanner instance.
-func NewScanner(db *database.DB, logger *logrus.Logger, minPinners int, serverName string, skydClient skyd.Client) *Scanner {
+func NewScanner(db *database.DB, logger *logrus.Logger, minPinners int, serverName string, customSleepBetweenScans string, skydClient skyd.Client) *Scanner {
+	sleep := sleepBetweenScans
+	if customSleepBetweenScans != "" {
+		hours, err := strconv.Atoi(customSleepBetweenScans)
+		if err != nil || hours < 1 {
+			logger.Warnf("Invalid value of PINNER_HOURS_BETWEEN_SCANS given, using the default of %d hours", sleepBetweenScans)
```

Comment:
I would return the error.  Setting invalid parameters should prevent a start up.